### PR TITLE
Make the attribute whitelist opt-out

### PR DESCRIPTION
**Pull request by Scott-Guest**
- Turn the whitelist attribute on by default (#2625)
- Add a flag `--no-pedantic-attributes` to disable the whitelist

### CODE REVIEW

**Comment by Scott-Guest** [comment]
Blocked until runtimeverification/k-exercises#37 is merged

**Review by dwightguth — APPROVED** [review]
Looks good. If you think you know how to handle it, it might be worth updating the text of the tutorial lesson. Otherwise seems great.

**Review by radumereuta — APPROVED** [review]

**Comment by dwightguth** [comment]
Part of #2625

**Pull request merged**
